views/index.html

```diff
@@ -1,15 +1,15 @@
 <div class="pure-g full-height-block dark-bg splash">
     <a href="/team"><div class="splash-container"></div></a>
     <div class="pure-u-1 splash-tagline">
-      <span>dwyl is a <span class="tagline-highlight">team</span> of people using <span class="tagline-highlight">technology</span> to solve real problems</span>
+      <span>dwyl /dwɪɫ/ is a <span class="tagline-highlight">worldwide community</span> of people using <span class="tagline-highlight">technology</span> to solve real problems.</span>
```

Comment:
@markwilliamfirth I'll leave it to you to decide whether this looks confusing or not:
<img width="571" alt="screen shot 2017-06-01 at 16 40 25" src="https://cloud.githubusercontent.com/assets/4185328/26687930/1a32b35a-46e9-11e7-9d50-84c069bce10e.png">